packages/block-editor/src/components/block-list/block-selection-button.native.js

```diff
@@ -38,11 +38,11 @@ const BlockSelectionButton = ( {
 			<TouchableOpacity
 				style={ styles.button }
 				onPress={ () => {
-					/* Open BottomSheet with markup */
+					/* Open BottomSheet with markup. */
```

Comment:
I think we have a different issue here. This is a single line comment so it should start with `//`.